tests/Agent/IntegrationTests/Applications/BasicMvcApplication/Controllers/CustomInstrumentationAsyncController.cs

```diff
@@ -57,7 +57,7 @@ public string GetBackgroundThreadWithError()
 
             // Intentionally run this method on a background thread so it won't be caught up as part of the current transaction.
             // Don't wait for it to finish or else it won't be able to generate a transaction trace (because it will be eclipsed by the request)
-            Task.Run(async () => await CustomMethodBackgroundThreadWithError()).Wait();
+            Task.Run(CustomMethodBackgroundThreadWithError);
```

Comment:
Actually, it may not be. It looks like the test that exercises this endpoint is failing. Still investigating.

Reply:
So the resolution for this is to put the code back like it was and modify how I'm making the call via HttpClient -- the original test fixture had an exception handler because WebClient would throw if there was an error making the remote call. My updates to use HttpClient were using a different method to make the call that didn't throw, thereby either failing in the response validation or requiring that I take the .Wait() off. 

I'm not convinced this test actually does what we want it to, but I've been able to put this controller back the way it was and get the test to pass like it did before. So I guess that's good enough.